_common

```diff
@@ -117,8 +118,55 @@ handle_unknown() {
         group_mailto=${group_mailto:-$notification_address}
         if [[ -n "$group_mailto" ]]; then
             group_name=$(echo "$group_data" | runjq -r '.[0].name')
-            echo "$header"$'\n'"Reason: $reason"$'\n'"$excerpt" | mailx -s "Unreviewed issue (Group $group_id $group_name)" -r "openqa-label-known-issues <$from_email>" "$group_mailto"
+            job_name=$(echo "$job_data" | runjq -r '.job.name')
+            job_result=$(echo "$job_data" | runjq -r '.job.result')
+            job_info="* Name: $job_name
+* Result: $job_result
+* Reason: $reason"
+            email="$header"$'\n'"$job_info"$'\n'$'\n'"$excerpt"
+            subject="Unreviewed issue (Group $group_id $group_name)"
+#            group_mailto="aaaaemro56u7c5ek5x4tkjs32q@suse.slack.com, timueller@suse.de"
```

Comment:
Disabled testing code.

Reply:
It's a **draft**

Reply:
I saw that, yes. Just was pointing out what I would point out later anyway to ensure it's not overlooked.